src/Foundatio.Repositories.Elasticsearch/Repositories/ElasticReadOnlyRepositoryBase.cs

```diff
@@ -714,6 +720,79 @@ protected virtual ICommandOptions<T> ConfigureOptions(ICommandOptions<T> options
         return options;
     }
 
+    protected virtual void ConfigureGetRequest(GetRequest request, ICommandOptions options)
+    {
+        var (resolvedIncludes, resolvedExcludes) = GetResolvedIncludesAndExcludes(options);
+
+        if (resolvedIncludes.Length > 0 && resolvedExcludes.Length > 0)
+        {
```

Comment:
This seems odd to have both includes and excludes.

Reply:
You can 100% have both.